<!-- Filename: icp_mapping.launch -->
<!-- Fileinfo: launch icp_mapping_node -->

<launch>
    <param name="use_sim_time" value="true" />
    <arg name="node_name" default="icp_mapping_node"/>
    <arg name="pkg_name" value="pcl_matching"/>
    <arg name="local" default="true"/>

    <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name)" output="screen" required="true">
        <rosparam  command="load" file="$(find pcl_matching)/config/config.yaml"/>
    </node>

</launch>
